docker-compose.yml

```diff
@@ -156,13 +156,19 @@ services:
       - "5432:5432"
     environment:
       POSTGRES_PASSWORD: mysecretpassword
+    networks:
+      vtto:
+        ipv4_address: 10.5.0.8
 
   adminer:
     container_name: adminer
     image: adminer
     restart: always
     ports:
       - 8888:8080
+    networks:
+      vtto:
+        ipv4_address: 10.5.0.9
```

Comment:
If noone is using this container we can actually remove it. I've only used it during the initial work at the db :)

Reply:
Are you referring to the `adminer`? I'll remove it if it's not being used at all.

Reply:
yes

Reply:
Done: https://github.com/get10101/10101/pull/285/commits/bc64ddc82ebdbb9a0ce8365a32078cfbb36066ec.